pandas/types/inference.py

```diff
@@ -82,25 +344,48 @@ def is_hashable(arg):
     >>> is_hashable(a)
     False
     """
-    # unfortunately, we can't use isinstance(arg, collections.Hashable), which
-    # can be faster than calling hash, because numpy scalars on Python 3 fail
-    # this test
+    # Unfortunately, we can't use isinstance(obj, collections.Hashable), which
+    # can be faster than calling hash. That is because numpy scalars on Python
+    # 3 fail this test.
 
-    # reconsider this decision once this numpy bug is fixed:
+    # Reconsider this decision once this numpy bug is fixed:
     # https://github.com/numpy/numpy/issues/5562
 
     try:
-        hash(arg)
+        hash(obj)
     except TypeError:
         return False
     else:
         return True
 
 
-def is_sequence(x):
+def is_sequence(obj):
+    """
```

Comment:
I think leaving both functions are fine for now, especially in the context of this PR.